api/agentpb/agent.proto

```diff
@@ -157,6 +157,7 @@ message StartActionRequest {
     string dsn = 1;
     string query = 2;
     repeated string values = 6;
+    string schema = 7;
```

Comment:
nit: shouldn't the keys be sorted in descending order?

Reply:
We have this on more places. Its order by logic order, not numeric one. To keep similar/related properties grouped. I can change it for sure if needed.